### ISSUE TITLE
[Refactor] Layout Components - Footer

### ISSUE THREAD

**Issue by aboydnw** [NASA-IMPACT/veda-ui#1135]
### Context

This ticket is for a layout component as part of the VEDA refactor. This components are used to generate the overall layout and site architecture of the Dashboard. They match the overall look and feel of the app and will be used in the template instance. Our recommendation will be to use these layout components, and we anticipate that most instances will use these components, but some may decide to bring their own.

Our footer implementation on [GHG](https://earth.gov/ghgcenter) is probably the most recent and complete implementation so far. This also depends on the design ticket #1123

This ticket might be broken down into smaller tickets after initial investigation.

**Notes:**
* Mark new files for VEDA2 refactor with `@VEDA2-REFACTOR-WORK`

**Acceptance Criteria:**
**General**
- [ ] Reusability => modularize stateless/presentational features to be as reuseable as possible within the core feature view containers
- [ ] Composite Feature should be entirely data agnostic => Separate out the data logic from this feature component into the data layer. _Note: Data will now have to be passed in as props so we will need to update the page views to supply this_
   - [ ] Composite Features should have stricter typed interfaces
- [ ] This feature should still support current instances and be backwards compatible

**Specific**
_{TO BE UPDATED}_

**Related to:** https://github.com/NASA-IMPACT/veda-ui/issues/801

**Resources:**
* ADR: https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/adr/002-application-architecture-for-configurability.md#decision-outcome
* [Miro Board outline Core Features](https://miro.com/app/board/uXjVNlIZVJI=/)

**Comment by snmln** [comment]
@aboydnw & @faustoperez I wanted to confirm are these the design that we should be working off of? 

**Design:** https://www.figma.com/design/PG7AhRloW6qJd9U00VdQ77/VEDA-Template-Instance?node-id=178-2&t=HAlyBRbfjH7jZgHy-1

**Comment by aboydnw** [comment]
Yes sorry I should have update this ticket with that most recent link. I believe most recent designs for the footer happened during this ticket: https://github.com/NASA-IMPACT/veda-ui/issues/1193
That has the same figma link that you shared @snmln 

@faustoperez correct me if that's wrong or if there is a better figma link

**Comment by faustoperez** [comment]
Please use the VEDA - USWDS Figma file for components, the one you list is the Template instance file

This is the correct link: https://www.figma.com/design/5mclPTReHcRIzKbJm8YA6a/VEDA---USWDS?node-id=473-586&t=dOX6eKbhgXYNuNu3-1

Thank you!

**Comment by faustoperez** [comment]
I have updated the VEDA-USWDS Figma file with the mobile footer designs 👉 https://www.figma.com/design/5mclPTReHcRIzKbJm8YA6a/VEDA---USWDS?node-id=473-586&t=NsDEcNxf1omkNZTF-1

<img width="1163" alt="Image" src="https://github.com/user-attachments/assets/2baafb01-348d-43c3-89e0-b091dab37da4" />
